tests/lambda_functions/va_profile/test_va_profile_integration.py

```diff
@@ -662,8 +737,50 @@ def test_va_profile_opt_in_out_lambda_handler_integration_testing(
     This unit test verifies that the lambda code attempts to load this certificate.
     """
 
+    # Must reload lambda module to properly mock ENV variables defined before running lambda handler
```

Comment:
I originally had the new ENV values mocked in the test, then I tried creating its own fixture. 

Since the environment variable are initialized when the module is loaded, the only way to repopulate the environment variables is to reload the module.